apps/Gruntfile.js

```diff
@@ -408,7 +414,7 @@ module.exports = function (grunt) {
   config.watch = {
     js: {
       files: ['src/**/*.{js,jsx}'],
-      tasks: ['newer:copy:src', 'exec:browserify', 'notify:browserify'],
+      tasks: ['newer:copy:src', 'exec:browserify', 'notify:browserify', 'exec:applabapi'],
```

Comment:
I'm not sure whether I'd prefer this before the notify step or to give it its own notification, but I think I want one or the other.